release-notes/major10/10.4.0/valtimo-frontend-libraries.md

```diff
@@ -33,6 +46,18 @@ The following bugs were fixed:
 
   Fixed an issue where the wrong assigneeFilter is active and displaying the wrong case list because of it.
 
+* **Fix IBM Carbon accent colors**
+
+  Fixed an issue where Carbon button and tag components were not themed with the Ritense blue theme colors, but with the
+  stock IBM blue instead. Refer to [this file](https://github.com/valtimo-platform/valtimo-frontend-libraries/blob/development/projects/valtimo/components/assets/css/carbon.scss)
+  if you want to customize theme colors in your implementation. Includes your custom theme file at the end of the
+  `styles` array in your implementation's `angular.json` instead of `carbon.scss` exported from `@valtimo/components`.
+
+* **Fix case list URL parameters**
+
+  Query parameters in the URL on the case list page (concerning pagination and search) were retained when navigating to
+  other pages inside the application. This issue has been resolved.
```

Comment:
I think the issue being listed here means it's been resolved, so I suggest removing that line.

```suggestion
  other pages inside the application.
```